src/main/java/net/smartcosmos/edge/things/service/UpdateThingEdgeServiceDefault.java

```diff
@@ -41,37 +48,46 @@ public UpdateThingEdgeServiceDefault(EventSendingService eventSendingService, Co
     @Override
     public void update(DeferredResult<ResponseEntity> response, String type, String urn, Map<String, Object> requestBody, SmartCosmosUser user) {
 
+        try {
+            response.setResult(updateWorker(type, urn, requestBody, user));
```

Comment:
use `e.toString()`.  It is better than `getMessage()`.

Create a message string to log with the `e.toString()` to provide context.log the type, response and urn.  It will make it easier to find the problem if it is data related or code related.